client/document/index.jsx

```diff
@@ -122,7 +122,7 @@ class Document extends React.Component {
 						[ 'is-group-' + sectionGroup ]: sectionGroup,
 						[ 'is-section-' + sectionName ]: sectionName,
 						'is-white-signup': sectionName === 'signup',
-						'is-mobile-app-view': app.isMobileApp,
+						'is-mobile-app-view': app && app.isWpMobileApp,
```

Comment:
Nit: we have optional chaining now 🙂 — `app?.isWpMobileApp`

Reply:
I second this. FWIW, what Jarda suggested will safely return `undefined` if `app` is not defined, which as a falsy value will just be ignored by `classNames()`.

Reply:
Fixed